common/modules/gcp-stackdriver-monitoring/main.tf

```diff
@@ -0,0 +1,125 @@
+terraform {
```

Comment:
Same, instead of duplicating the whole module, what about doing this with native TF?

Reply:
will do later in a separate task/ticket, I'm looking forward to close this particular task.